docs/gaming/partial-sign-in-backend.md

```diff
@@ -0,0 +1,105 @@
+---
+title: How to sign a transaction in the backend
```

Comment:
Separate PR please. Also not part of the gaming section, move to offline transactions under main cookbook